src/server/node_services/nodes_monitor.js

```diff
@@ -228,8 +228,9 @@ class NodesMonitor extends EventEmitter {
             );
             nodes_stats.forEach(stats => {
                 dbg.log0(`resetting stats in prometheus:`, stats);
-                prom_report.instance().set_providers_ops(stats.service, stats.write_count, stats.read_count);
-                prom_report.instance().set_providers_bandwidth(stats.service, stats.write_bytes, stats.read_bytes);
+                const core_report = prom_reporting.get_report(config.NOOBAA_CORE_METRICS_REPORT);
```

Comment:
If you are using the same report twice, maybe it is better to require if so you won't get in multiple times from `prom_reporting.get_report `?

Reply:
1. It is a bad practice to add code to require as it will make it much harder to move native imports.
2. we can move this to global scope but I don't see the value. It is like advocating that in the old code we should not call `instance()` more than once.